tidy3d/components/medium.py

```diff
@@ -2927,6 +2927,51 @@ class CustomAnisotropicMediumInternal(CustomAnisotropicMedium):
     )
 
 
+class AbstractNonlinearMedium(AbstractMedium, ABC):
+    """Abstract nonlinear medium.
+
+    Note
+    ----
+    The nonlinear constitutive relation is solved iteratively; it may not converge
+    for large nonlinearities. Increasing `numiters` can help with convergence.
+
+    """
+
+    numiters: pd.PositiveInt = pd.Field(
+        1,
```

Comment:
is 1 a reasonable number or should this be adjusted case by case? it might be good to start with a default value of something more practical for most cases (define as a constant at the top of file)? but maybe I'm not understanding something.

Reply:
1 has been working fine for me. I think it might need to be adjusted if the nonlinearity was quite strong.